cli/roles.go

```diff
@@ -0,0 +1,27 @@
+package main
```

Comment:
This exists because we expected users to be wondering where to get their roles, but it proved too technically challenging to implement at the time. Instead of leave users guessing, we decided to put in a command that would at the very least direct them to us.